src/api.js

```diff
@@ -205,10 +208,10 @@ async function getUserMessages(userId) {
     method: 'GET',
     uri: process.env.API_URL + '/clients/' + userId + '/messages',
     headers: {
-      Authorization: 'Bearer ' + process.env.OAUTH_ACCESS_TOKEN
+      Authorization: 'Bearer ' + process.env.OAUTH_ACCESS_TOKEN,
```

Comment:
Use template interpolation instead of concatenation i.e `Bearer ${process.env.OAUTH_ACCESS_TOKEN}`